docs/parameters.yaml

```diff
@@ -2496,6 +2496,36 @@ type: duration
 default: 360h
 components: ["origin", "cache", "director", "registry"]
 ---
+name: Monitoring.LabelLimit
+description: |+
+  The maximum number of labels that can be attached to a single metric. 0 means no limit.
+type: int
+default: 64
```

Comment:
These defaults don't get set automatically by adding them here. You'll want to add them all to `config/resources/defaults.yaml` or in `config/config.go::InitServerDefaults()`. I think the defaults.yaml is probably the better place in this case because the defaults are simple ints and don't depend on the detected runtime environment.